pkg/cli/root.go

```diff
@@ -598,10 +598,18 @@ func validateSync(config *config.Config) error {
 				for _, content := range regCfg.Content {
 					ok := glob.ValidatePattern(content.Prefix)
 					if !ok {
-						log.Error().Err(glob.ErrBadPattern).Str("pattern", content.Prefix).Msg("sync pattern could not be compiled")
+						log.Error().Err(glob.ErrBadPattern).Str("prefix", content.Prefix).Msg("sync prefix could not be compiled")
 
 						return glob.ErrBadPattern
 					}
+
+					if content.StripPrefix && !strings.Contains(content.Prefix, "/*") && content.Destination == "/" {
+						log.Error().Err(errors.ErrBadConfig).
+							Interface("sync content", content).
+							Msg("sync config: can not use stripPrefix true and destination '/' without using glob patterns in prefix")
```

Comment:
given a image named /a/b/c
and sync content config:
```
			{
				content:  sync.Content{Prefix: "/a/b/c", Destination: "/", StripPrefix: true},
			}
```

It will put /a/b/c directly into zot-storage.RootDir(), because repoName would be "/", and not under zot-storage.RootDir()/c, so it will break things in zot.

StripPrefix should be use with a valid destination or when using globaPatterns like /a/b/* and it will only strip /a/b/ without c